model/license.go

```diff
@@ -16,6 +17,10 @@ const (
 	LICENSE_RENEWAL_LINK  = "https://mattermost.com/renew/"
 )
 
+var (
+	trialDuration = (time.Hour * 24 * 30) + (time.Hour * 8)
```

Comment:
A little more clear operation plus comment to explain won't hurt
```suggestion
	trialDuration = 30 * (time.Hour * 24) + (time.Hour * 8) // 720 hours (30 days) + 8 hours is trial license duration
```

Reply:
Done.